index.html

```diff
@@ -0,0 +1,157 @@
+            <ul class="wide-nav-bar-ul">
+                <li class="wide-nav-li">
+                    <a href="#" class="wide-nav-link" >Home</a>
+                </li>
+                <li class="wide-nav-li">
+                    <a href="about.html" class="wide-nav-link" >About</a>
+                </li>
+                <li class="wide-nav-li">
+                    <a href="#main-program-container" class="wide-nav-link" >Program</a>
+                </li>
+                <li class="wide-nav-li">
+                    <a href="#featured-speaker-container" class="wide-nav-link">Speakers</a>
+                </li>
+            </ul>
+        </nav>
+        <!-- hamburger -->
+        <nav class="nav-section">
+            <a href="#" id="hamburger"><img src="img/hamburger.png" alt="menu icon" class="hamburger-img" /></a>
+        </nav>
+        <!-- mobile nav -->
+        <nav class="mobile-nav">
+            <a id="mobile-drawer-close-icon">
+                <img src="img/close_icon.png" alt="Ext icon" class="close-icon">
+            </a>
+            <ul class="mobile-nav-ul">
+                <li class="mobile-nav-li">
+                    <a href="#" class="mobile-nav-link"><img src="img/cc_logo_white.png" alt="logo" class="mobile-nav-img"></a>
+                </li>
+                <li class="mobile-nav-li menu-items">
+                    <a href="#" class="mobile-nav-link" id="home-link">
+                        <div class="mobile-nav-element-container">
+                            <h3 class="mobile-nav-link-title">Home</h3>
+                            <img src="img/icon_next.png" alt="" class="mobile-nav-icon-next">
+                        </div>
+                    </a>
+                </li>
+                <li class="mobile-nav-li">
+                    <a href="about.html" class="mobile-nav-link" id="about-link">
+                        <div class="mobile-nav-element-container">
+                            <h3 class="mobile-nav-link-title">About</h3>
+                            <img src="img/icon_next.png" alt="" class="mobile-nav-icon-next">
+                        </div>
+                    </a>
+                </li>
+            </ul>
+        </nav>
+
+    </header>
+
+    <main>
+        <div class="welcome-container">
+            <h2 class="welcome-subtitle">Hello Sharing World</h2>
+            <h1 class="welcome-title">CREATIVE COMMONS GLOBAL SUMMT 2022</h1>
+            <div class="welcome-description-container">
+                <p class="welcome-description-text">
+                    A joyfull celebration believing in the value of openness and sharing.
+                    creating a positive change with people from over 80 place taking place in October in Korea.
+                </p>
+            </div>
+            <div class="welcome-date-adress-container">
+                <h1 class="welcome-date-txt">2015.10.15(THU) ~ 16(FRI)</h1>
```

Comment:
- **Please, Here's how Hn tags should be used:** For an article or webpage, remember that the H1 title is the most important section. H2 and H3 are used to organize sub-sections, while H4, H5, and H6 are intended to provide additional information, with more details.
- Kindly Fix the use of `h1` in both inde.html and About.Html